cnext_app/lib/components/code-panel/monaco/CodeEditor.tsx

```diff
@@ -310,6 +312,21 @@ const CodeEditor = ({ stopMouseEvent }) => {
         }
     });
 
+    useEffect(() => {
+        if (monaco) {
+            const path = store.getState().projectManager.activeProject?.path;
```

Comment:
I don't think this will be called when activeProject changed.

Reply:
the Editor will re-render again if we change the activate the project right?
I was tested with the same name of the file. It still working well @mbui0327